packages/next/build/webpack-config.ts

```diff
@@ -139,11 +139,9 @@ export function getDefineEnv({
         }),
     // TODO: enforce `NODE_ENV` on `process.env`, and add a test:
     'process.env.NODE_ENV': JSON.stringify(dev ? 'development' : 'production'),
-    ...((isNodeServer || isEdgeServer) && {
-      'process.env.NEXT_RUNTIME': JSON.stringify(
-        isEdgeServer ? 'edge' : 'nodejs'
-      ),
-    }),
+    'process.env.NEXT_RUNTIME': JSON.stringify(
+      isEdgeServer ? 'edge' : isNodeServer ? 'nodejs' : 'browser'
```

Comment:
We had a [discussion on slack](https://vercel.slack.com/archives/C02CDC2ALJH/p1650587234077569) that we don't want to add `"browser"` as a value here

Reply:
replaced with undefined!